provisioner/terraform/executor_unix.go

```diff
@@ -0,0 +1,19 @@
+//go:build !windows
+
+package terraform
+
+import (
+	"context"
+	"os"
+	"os/exec"
+)
+
+func interruptCommandOnCancel(ctx, killCtx context.Context, cmd *exec.Cmd) {
+	go func() {
+		select {
+		case <-ctx.Done():
+			_ = cmd.Process.Signal(os.Interrupt)
```

Comment:
Should we use build tags for this? Since there are no OS-specific imports, I'd assume idiomatically we'd just check if `runtime.GOOS == "windows"`.